nifi-nar-bundles/nifi-extension-utils/nifi-database-utils/src/test/java/org/apache/nifi/util/db/TestJdbcClobReadable.java

```diff
@@ -83,9 +80,10 @@ public void testDriverLoad() throws ClassNotFoundException {
     }
 
     private void validateClob(String someClob) throws SQLException, ClassNotFoundException, IOException {
-        folder.delete();
-
-        final Connection con = createConnection(folder.getRoot().getAbsolutePath());
+        File folder = Files.createTempDirectory(String.valueOf(System.currentTimeMillis()))
+                .resolve("db")
+                .toFile();
+        final Connection con = createConnection(folder.getAbsolutePath());
```

Comment:
See `DBCPServiceTest` for handling `derby.log` and temporary database directory lifecycle.

Reply:
The updates to set the `derby.log` look good, but this test class still needs a method to clear the database directory after each method execution.